packages/hash/api/src/graph/system-types.ts

```diff
@@ -68,59 +62,54 @@ export let SYSTEM_TYPES: {
      */
     dummy: EntityTypeModel;
   };
-  /**
-   * @todo: bring back link types as entity types
-   * @see https://app.asana.com/0/1202805690238892/1203361844133477/f
-   */
-  // linkType: {
-  //   // HASHInstance-related
-  //   admin: LinkTypeModel;
+  entityLinkType: {
+    // HASHInstance-related
+    admin: EntityTypeModel;
 
-  //   // User-related
-  //   hasMembership: LinkTypeModel;
+    // User-related
+    hasMembership: EntityTypeModel;
 
-  //   // OrgMembership-related
-  //   ofOrg: LinkTypeModel;
+    // OrgMembership-related
+    ofOrg: EntityTypeModel;
```

Comment:
These two can be consolidated into a single "membership" link entity type now which can contain a `responsibility` property type and such. If you want we can chat about this

Reply:
My instinct is this would make more sense to do this in a standalone PR once we have linting/type-checking passing again, as this change would cascade into the FE. I've created a task for it earlier today here: https://app.asana.com/0/0/1203371754468058/f

Reply:
We spoke about this internally, have left a `@todo` so we can address it in follow-up in https://github.com/hashintel/hash/pull/1400/commits/181bf12bc72d38ac49fc7979a980a82962effbc2